### ISSUE TITLE
Grid color property is white

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-next#1142]
There appears to be a color property for `Grid` and `GridItem` patternfly-next? Is that necessary?

I encountered this issue while adding a bullet chart to our Cost Management app. The text (labels, legend, etc.) are all white. Looking at the HTML elements for `Grid` and `GridItem`, I see that the color is white. 

I can confirm this with the examples for patternfly-react as well as patternfly-core. Although, I don't see a color property in grid.scss?

Before:
![before](https://user-images.githubusercontent.com/17481322/50291147-bce1b180-043b-11e9-92de-caffedd9d071.png)

When I deactivate the color style, I can see the text in my bullet chart. Also, if I don't place the chart in the `GridItem`, I can see the text.

After:
![after](https://user-images.githubusercontent.com/17481322/50291212-f3b7c780-043b-11e9-99e9-eff59935e394.png)

My workaround is to set the color manually, but prefer not to do that.

If a color property is necessary, perhaps white text was intended for the dark theme?

**Comment by mcoker** [comment]
@dlabaj is the class `.pf-t-dark` anywhere on a parent here?

**Comment by mcoker** [comment]
@dlabrecq hmm, I spun up my local server and am not seeing that.

<img width="1854" alt="screen shot 2018-12-21 at 3 38 08 pm" src="https://user-images.githubusercontent.com/35148959/50364638-758c1b80-0536-11e9-9fb5-8f3311aaedea.png">

What happens is in `.pf-t-dark` and when a component includes the `%pf-t-dark` sass placeholder, this variable change happens:

`--pf-global--Color--100: var(--pf-global--Color--light-100); // #fff`

Have you included `.pf-t-dark` on a parent node? If not is there a branch or something I can pull down to reproduce this?

**Comment by dlabrecq** [comment]
We only set pf-m-dark-100 on the overview page, not for the detail pages.

You're probably not seeing it locally because I already added a workaround to override the color on the bullet chart back to black. 

Just below the pf-l-grid__item selector, you'll see a div that wraps the bullet chart. This has an odd selector name (e.g., css-b5nuhj), which is where I override the color.

![screen shot 2018-12-21 at 8 55 13 pm](https://user-images.githubusercontent.com/17481322/50369379-bf8af680-0562-11e9-9062-b840e05ace69.png)

I can also see the color setting on both pf-l-grid with the patternfly core examples as well.

![screen shot 2018-12-21 at 8 50 43 pm](https://user-images.githubusercontent.com/17481322/50369346-29ef6700-0562-11e9-9af1-2ae5e4754bdc.png)
